src/index.js

```diff
@@ -1 +1,47 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { getAPIScores, setAPIScores } from './modules/hit-api.js';
+import { createScoreListElement, appendListElement, clearForms } from './modules/scores.js';
+import Score from './modules/singleScore.js';
+
+const addToLocalStorage = (scores) => {
+  localStorage.setItem('scores', JSON.stringify(scores));
+};
+
+const addScoreList = document.getElementById('add');
+const refreshScoreList = document.getElementById('refresh');
+
+const scores = new Score();
+
+addScoreList.addEventListener('click', () => {
+  const newName = document.getElementById('user').value;
+  const newScore = document.getElementById('score').value;
+
+  setAPIScores(newName, newScore);
+
+  const listElement = createScoreListElement(newName, newScore);
+
+  appendListElement(listElement);
+  addToLocalStorage();
+  clearForms();
+});
+
+const updateLocalStorage = (scores) => {
+  appendListElement(scores);
+  localStorage.clear();
+  addToLocalStorage(scores);
+};
+
+const fromLocalStorage = () => {
+  if (localStorage.length !== 0) {
+    const scoresStored = JSON.parse(localStorage.getItem('scores'));
+    scoresStored.listElement.forEach((user) => {
+      scores.listElement.push(user);
+    });
+    updateLocalStorage(scores);
+  }
+  refreshScoreList();
+};
+
+fromLocalStorage();
+
+refreshScoreList.addEventListener('click', () => getAPIScores());
```

Comment:
- [ ] It looks like there is an error when you fetch the data from the API in your app. When I run the app, it shows an empty page, with no names or scores. After submitting a name and a score, I can see the result on the screen, as expected, but when I refresh the page, the result I added is also gone. Probably there is something going on with your request function. Please kindly take a look at it.
_Note: I know the `POST` request is working because I took a look at the API and there is the test name that I included, so it's only the `GET` request the one that is giving some problems._
Here are a couple of images to show you what I'm saying:
- This is the test name that I used, as you can see, it is in the API results.
![Captura de pantalla 2022-02-04 192832](https://user-images.githubusercontent.com/59100477/152583939-a77d9f40-d1f4-492a-af80-48554c6d52c2.jpg)
- But after refreshing the page this is what I get.
![Captura de pantalla 2022-02-04 192858](https://user-images.githubusercontent.com/59100477/152584044-9f95210f-ff3d-4eee-9ece-968501a6222b.jpg)


- [ ] Also, I can see that the `refresh` button that you included, it's not doing anything, when I click nothing happens, kindly take a look.